### DISCUSSION TITLE
Request to delete account on Jira

### DISCUSSION THREAD

**Discussion by vsoch** [apache/airflow#13849] [Q&A]
Happy New Year airflow community! I am cleaning up old accounts, and would like to request deletion of user [vsoch](https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-4355?filter=allopenissues) on the JIRA airflow board. I looked to try and create an issue there, but it looks like the project isn't in the list anymore. Please let me know what you need me to do, and thank you!

**Comment by ashb** [comment]
I've opened an issue for you on the Infra project on Jira - https://issues.apache.org/jira/browse/INFRA-21388

**Comment by gmcdonald** [comment]
We can not delete the account as it has reported and been assigned a ticket. But I can disable it
